examples/platform/silabs/wfx_rsi.h

```diff
@@ -28,9 +30,8 @@
  * Interface to RSI Sapis
  */
 
-#define WFX_RSI_WLAN_TASK_SZ (1024 + 512 + 256) /* Stack for the WLAN task	 	*/
-#define WFX_RSI_TASK_SZ (1024 + 1024)           /* Stack for the WFX/RSI task		*/
-#define WFX_RSI_BUF_SZ (1024 * 10)              /* May need tweak 			*/
+#define WFX_RSI_WLAN_TASK_SZ (1024 + 512 + 256) /* Stack for the WLAN task	 	*/ // TODO: For rs9116
+#define WFX_RSI_BUF_SZ (1024 * 10) /* May need tweak 			*/                 // TODO: For rs9116
```

Comment:
`/* May need tweak */ `May need tweak comment as been there for a while now. Is it the right value or not? can see remove the comment

Reply:
removed the comment